### PR TITLE
 Doc for KFP v2 component I/O: Fix non-functioning example

### PR DESCRIPTION
**Pull request by jacobmalmberg**
The example contained a variable that does not exist. I suggest improving the line by replacing it with an f string without the missing variable. I have tested it and it works.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: content/en/docs/components/pipelines/sdk/v2/v2-component-io.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ def train(
 
     with open(model.path, 'w') as output_file:
         for i in range(num_steps):
-            output_file.write('Step {}\n{}\n=====\n'.format(i, line))
+            output_file.write(f"Step {i}\n=====\n")
```
Comment (chensun) [inline content/en/docs/components/pipelines/sdk/v2/v2-component-io.md:116]: nit: use single quotes for consistency within the file.

**Review by chensun — COMMENTED** [review]
/lgtm

File: content/en/docs/components/pipelines/sdk/v2/v2-component-io.md (hunk 1 above)
Comment (jacobmalmberg) [reply content/en/docs/components/pipelines/sdk/v2/v2-component-io.md]: good catch. i've pushed a new commit that fixes this

**Comment by jacobmalmberg** [comment]
The merge conflict has been resolved. I made some mistakes using the rebase feature of git. This should now have been fixed. Apologies..

**Pull request merged**